app/admin/emails.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register Email do
+  # See permitted parameters documentation:
+  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+  #
+  # Uncomment all parameters which should be permitted for assignment
+  #
+
+  permit_params :email_type, :subject, :body
+
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:email_type, :subject, :body]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
+end
```

Comment:
Are all of these comments supposed to be here? The same thing for the other files that have comments. I don't think this adds any valuable information.

Reply:
No, I'll delete them